### PR TITLE
Search functionality

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
Initial implementation of search functionality. At the moment it doesn't provide much other than the following endpoints:

* GET all profiles
* GET profiles based on query
* POST object to index

**Comment by johngian** [comment]
@fiji-flo r?

File: dinopark_search/views.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+
+@blueprint.route('/profiles/index', methods=['POST'])
+def index_profile():
+    """Index an object to ES."""
+
+    obj = request.get_json()
+    es = get_es_client()
+    result = es.index(
+        index=settings.ES_WEBCOMPAT_INDEX,
```
Comment (fiji-flo) [inline dinopark_search/views.py]: I guess that should be `ES_DINOPARK_INDEX`

File: docker/prod.Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+# Copy Pipfile related files
+COPY Pipfile.lock /code/
+COPY Pipfile /code/
+
+# Install dependencies
+RUN pip install pipenv && pipenv install --system --deploy
+
+# Default command to run app using gunicorn wsgi server
+CMD gunicorn webcompat_search:create_app\(\) -b 0.0.0.0:${PORT:-5000} -w 3
```
Comment (fiji-flo) [inline docker/prod.Dockerfile]: `s/webcompat/dinopark/`

**Pull request merged**